binrz/man/rizin.1

```diff
@@ -78,84 +82,87 @@ List supported IO plugins
 Map file at given address
 .It Fl M
 Disable demangling
-.It Fl n, nn
+.It Fl n , nn
 Control loading of RzBin info, with -nn only loading binary structures
 .It Fl N
-Disable user settings/projects from ~/.rizinrc, ~/.config/rizin/rizinrc and the scripts inside .../rizinrc.d/ directory
+Disable user settings from ~/.config/rizin/rizinrc. Check RZ_CONFIGHOME in rizin -H output.
 .It Fl NN
 Disable loading any scripts or plugins
 .It Fl q
 Quiet mode, suppresses prompt and quits after script execution
 .It Fl qq
-Quit before showing the prompt. Right after all the -e -c and -i arguments are evaluated
+Quit before showing the prompt.
+Right after all the -e -c and -i arguments are evaluated.
 .It Fl Q
```

Comment:
The `-Q` flag is still there in the man file -- please remove it as @notxvilka mentioned.